src/System.Net.Sockets/src/System/Net/Sockets/UdpReceiveResult.cs

```diff
@@ -21,32 +26,32 @@ public UdpReceiveResult(byte[] buffer, IPEndPoint remoteEndPoint)
                 throw new ArgumentNullException(nameof(remoteEndPoint));
             }
 
-            _buffer = buffer;
```

Comment:
Why change this?  Unless there is a side-affect when setting properties vs. fields, it is more performant to just set the field.

Reply:
@davidsh Good to know that using fields in this cases are better. I was expecting to be the same.

@stephentoub I miss the default() case, my fault.

Reverting first commit. Only left docs